doorip-api/src/main/java/org/doorip/trip/service/TripDetailService.java

```diff
@@ -209,10 +209,11 @@ private boolean isEqualUserAndParticipantUser(User user, User participantUser) {
     }
 
     private int getValidatedResult(User user) {
-        try {
-            return user.getResult().getNumResult();
-        } catch (NullPointerException e) {
+        if (user.getResult() == null) {
             return -1;
         }
+        else {
+            return user.getResult().getNumResult();
+        }
```

Comment:
```suggestion
        if (user.getResult() == null) {
            return -1;
        }
        return user.getResult().getNumResult();
```
early return을 활용해봐도 좋을 것 같습니다~
그리고 -1을 문자열 상수로 정의해서 사용해도 좋을 것 같습니다.